json-rpc/src/methods.rs

```diff
@@ -164,5 +170,8 @@ pub(crate) fn build_registry() -> RpcRegistry {
     register_rpc_method!(registry, get_transactions, 3);
     register_rpc_method!(registry, get_account_transaction, 3);
     register_rpc_method!(registry, get_events, 3);
+
```

Comment:
nit: remove whitespace?

Reply:
it's intentional to separate second layer of APIs (with proofs)